src/sentry/testutils/cases.py

```diff
@@ -3391,30 +3397,7 @@ def load_trace(self) -> None:
         )
 
         # First Generation
-        # TODO: temporary, this is until we deprecate using this endpoint without useSpans
-        self.gen1_span_ids = self._generate_span_ids()
-        self.gen1_project = self.create_project(organization=self.organization)
-        self.gen1_events = [
-            self.create_event(
-                trace_id=self.trace_id,
-                transaction=f"/transaction/gen1-{i}",
-                spans=[
-                    {
-                        "same_process_as_parent": True,
-                        "op": "http",
-                        "description": f"GET gen2-{i}",
-                        "span_id": gen1_span_id,
-                        "trace_id": self.trace_id,
-                    }
-                ],
-                parent_span_id=root_span_id,
-                project_id=self.gen1_project.id,
-                milliseconds=2000,
-            )
-            for i, (root_span_id, gen1_span_id) in enumerate(
-                zip(self.root_span_ids, self.gen1_span_ids)
-            )
-        ]
+        self.populate_project1()
```

Comment:
In `load_errors` we need access to project1, thus, refactoring it into its function.
<img width="677" alt="image" src="https://github.com/getsentry/sentry/assets/44410/c455d4ed-6713-43ea-b42b-eb3c141fe088">